spec/api-browser-window-spec.js

```diff
@@ -1273,8 +1273,12 @@ describe('BrowserWindow module', function () {
         const initialWebContents = webContents.getAllWebContents().map((i) => i.id)
         ipcRenderer.send('prevent-next-new-window', w.webContents.id)
         w.webContents.once('new-window', () => {
-          assert.deepEqual(webContents.getAllWebContents().map((i) => i.id), initialWebContents)
-          done()
+          // We need to give it some time so the windows get properly disposed (at least on OSX).
+          setTimeout(() => {
```

Comment:
@zcbenz the timeout is no longer needed after somebody fixed the test by adding the  the map((i) => i.id), so feel free to remove it.

Reply:
That change was to make it easy to debug the problem, this test had still been very flaky.